aten/src/ATen/native/mkldnn/xpu/detail/QConv.cpp

```diff
@@ -54,7 +66,7 @@ at::Tensor quantized_convolution(
     at::Tensor weight,
     at::Tensor weight_scales,
     at::Tensor weight_zero_points,
-    std::optional<at::Tensor> bias,
+    c10::optional<at::Tensor> bias,
```

Comment:
Use `std::optional` here.

Reply:
Thanks for your tip, I have modified the all `c10` back to `std` in Qconv.cpp and Attr.h